heon/qdisc_test.sh

```diff
@@ -55,34 +55,35 @@ run "default: pfifo_fast"
 
 setup
 
-
-# set up a netem qdisc with delay to ensure setup works properly
+# apply delay & reordering to the packets 
```

Comment:
I'm wondering whether it could help to run just one qdisc and reset, rather than running all of them in sequence.
The qdisc of choice can come from a command line argument. 
You could then show how to use the script with two specific qdiscs, a FIFO, and then a `netem` with `reorder` in the README.

Reply:
That's a good idea. I have modified the test script to take the qdisc name as an argument.